docs/topics/whatsnew-eap.md

```diff
@@ -352,6 +462,121 @@ kotlin.build.archivesTaskOutputAsFriendModule=false
 
 For more information, see the issue in [YouTrack](https://youtrack.jetbrains.com/issue/KT-69330).
 
+### Option to share JVM artifacts between projects as class files
+> This feature is Experimental. It may be dropped or changed at any time. Use it only for evaluation purposes. We would appreciate your feedback on it in [YouTrack]()https://youtrack.jetbrains.com/issue/KT-61861/Gradle-Kotlin-compilations-depend-on-packed-artifacts. Opt-in is required (see details below).
```

Comment:
```suggestion
> This feature is [Experimental](components-stability.md#stability-levels-explained). It may be dropped or changed at any time. Use it only for evaluation purposes. We would appreciate your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issue/KT-61861/Gradle-Kotlin-compilations-depend-on-packed-artifacts). Opt-in is required (see details below).
```